### PR TITLE
Add unit tests for model messages

### PR DESCRIPTION
**Pull request by spivachuk**

### CODE REVIEW

File: x/model/types/messages_model_version_test.go [hunk 1]
```diff
@@ -1,74 +1,736 @@
...
+				msg.ReleaseNotesUrl = ""
+				return msg
+			}(validMsgCreateModelVersion()),
+		},
+		{
+			name: "ReleaseNotesUrl length == 256",
+			msg: func(msg *MsgCreateModelVersion) *MsgCreateModelVersion {
+				msg.ReleaseNotesUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(256-30) // length = 256
+				return msg
+			}(validMsgCreateModelVersion()),
```
Comment (ashcherbakov) [inline x/model/types/messages_model_version_test.go:423]: Please add the following positive cases:
- SoftwareVersion=0
- MinApplicableSoftwareVersion=0 and MaxApplicableSoftwareVersion=0
- MinApplicableSoftwareVersion=0 and MaxApplicableSoftwareVersion>0

File: x/model/types/messages_model_version_test.go [hunk 2]
```diff
@@ -1,74 +1,736 @@
...
+				msg.ReleaseNotesUrl = ""
+				return msg
+			}(validMsgUpdateModelVersion()),
+		},
+		{
+			name: "ReleaseNotesUrl length == 256",
+			msg: func(msg *MsgUpdateModelVersion) *MsgUpdateModelVersion {
+				msg.ReleaseNotesUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(256-30) // length = 256
+				return msg
+			}(validMsgUpdateModelVersion()),
```
Comment (ashcherbakov) [inline x/model/types/messages_model_version_test.go:684]: Please add the following positive case:
- SoftwareVersion=0

File: x/model/handler_test.go [hunk 3]
```diff
@@ -401,6 +401,140 @@ func TestHandler_PartiallyUpdateModelVersion(t *testing.T) {
...
+	// query updated model version
+	receivedModelVersion, err = queryModelVersion(
+		setup,
+		msgUpdateModelVersion.Vid,
+		msgUpdateModelVersion.Pid,
+		msgUpdateModelVersion.SoftwareVersion,
+	)
+	require.NoError(t, err)
+
+	// check that min version has been updated
```
Comment (ashcherbakov) [inline x/model/handler_test.go:533]:
```suggestion
	// check that max version has been updated
```

File: x/model/types/messages_model_test.go [hunk 4]
```diff
@@ -1,104 +1,1064 @@
...
+				msg.ProductUrl = ""
+				return msg
+			}(validMsgCreateModel()),
+		},
+		{
+			name: "ProductUrl length == 256",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.ProductUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(256-30) // length = 256
+				return msg
+			}(validMsgCreateModel()),
```
Comment (ashcherbakov) [inline x/model/types/messages_model_test.go:471]: Please add the following positive tests (for create and update messages):
- CommissioningModeInitialStepsHint=0
- CommissioningModeSecondaryStepsHint=0

File: x/model/types/messages_model_version_test.go (hunk 1 above)
Comment (spivachuk) [reply x/model/types/messages_model_version_test.go:466]: Done.

File: x/model/types/messages_model_version_test.go (hunk 2 above)
Comment (spivachuk) [reply x/model/types/messages_model_version_test.go:741]: Done.

File: x/model/handler_test.go (hunk 3 above)
Comment (spivachuk) [reply x/model/handler_test.go]: Fixed.

File: x/model/types/messages_model_test.go (hunk 4 above)
Comment (spivachuk) [reply x/model/types/messages_model_test.go:499]: Added the tests for `MsgCreateModel`.
As to `MsgUpdateModel`, it does not have the specified fields.

**Review by ashcherbakov — APPROVED** [review]

**Pull request merged**